Goal: amavis is only suggested, so disable by default

Author: Mathieu Parent <math.parent@gmail.com>

Upstream status: Specific to Debian

Index: b/kolab_bootstrap.in
===================================================================
--- a/kolab_bootstrap.in
+++ b/kolab_bootstrap.in
@@ -517,7 +517,7 @@
         'postfix-mynetworks' => $mynetworkinterfaces,
         #'postfix-relayhost' => "",
         #'postfix-transport' => "",
-        'postfix-enable-virus-scan' => "TRUE",
+        'postfix-enable-virus-scan' => "FALSE",
         'cyrus-autocreatequota' => 100000,
         'cyrus-quotawarn' => 80,
         'cyrus-admins' => "manager",
